Update images.md (#1090)
elastic/docs-builder: docs/syntax/images.md

# Images

Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with the following differences:

1. instead of link text, you provide an image description
2. an image description starts with `![` not just `[`

> 3. image paths must be specified as absolute paths

Author: ```suggestion
3. there are restrictions on the scope of image paths
```

:::{note}

> If you reference an image using a relative path, it must be relative to the `toc.yml` or `docset.yml` rather than relative to the page that uses it.

Reviewer: It can be a relative path but it can not go relative beyond the toc. 

e.g 

```
/images/
  - other.png
/solutions/
   /images/
      - observability-apm-app-landing.png`
   /page/
      - index.md
   toc.yml
```

`../images/observability-apm-app-landing.png` from `index.md` is fine because that lives in the same subtree as the `toc.yml` the page lives in. 

`../../images/other.png` lives outside the `toc.yml` subtree so is at risk of being broken if the assembler rehomes this subtree.

`

Author: Thanks, i've taken another go at the explanation since i think this might confuse folks

> If you reference an image using a relative path, it must be relative to the `toc.yml` or `docset.yml` rather than relative to the page that uses it.

Author: ```suggestion
If a page uses a relative image path, it must be relative to the `toc.yml` or `docset.yml` that includes the page rather than relative to the page itself.
```

> If a page uses an image that exists outside the folder that contains the `toc.yml` file or `docset.yml` file that contains that page, the image will fail to render and will generate warnings.

Author: ```suggestion
If a page uses an image that exists outside the folder that contains the `toc.yml` file or `docset.yml` file that contains that page, the image will fail to render and will generate warnings. Likewise, if a snippet in a [file inclusion](/syntax/file_inclusion.md) includes an image and is used in pages that exist in different `toc.yml`, the images will break.
```

> When a page is referenced in a `toc.yml` file or a `docset.yml` file and you reference an image that exists outside the tree represented in that `.yml` file, it will fail to render and will generate warnings.

Author: ```suggestion
If a page uses an image that exists outside the folder that contains the `toc.yml` file or `docset.yml` file that contains that page, the image will fail to render and will generate warnings.
```

> Irrespective of whether you use relative or absolute paths, if you reference an image that exists in a folder that is higher in the tree than the `toc.yml` file, it is at risk of being broken if the assembler rehomes this subtree and causes a warning in the build.

Author: ```suggestion
When a page is referenced in a `toc.yml` file or a `docset.yml` file and you reference an image that exists outside the tree represented in that `.yml` file, it will fail to render and will generate warnings.
```

> This typically fails to render when you run a local build, thus it's preferrable to use an absolute path, for example: `/solutions/images/observability-apm-app-landing.png`.

Author: ```suggestion
This type of relative path may fail to render when you run a local build, thus it's generally preferable to use an absolute path, for example: `/solutions/images/observability-apm-app-landing.png`.
```

> This type of relative path may fail to render when you run a local build, thus it's generally preferable to use an absolute path, for example: `/solutions/images/observability-apm-app-landing.png`.

Author: ```suggestion
This type of relative path may fail to render when you run a local build, thus it's generally preferable to use an absolute path, for example: `/solutions/images/observability-apm-app-landing.png`.

Irrespective of whether you use relative or absolute paths, if you reference an image that exists in a folder that is higher in the tree than the `toc.yml` file, it is at risk of being broken if the assembler rehomes this subtree and causes a warning in the build.
```

# Images

Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with the following differences:

1. instead of link text, you provide an image description
2. an image description starts with `![` not just `[`
3. there are restrictions on the scope of image paths

:::{note}

> This type of relative path may fail to render when you run a local build, thus it's generally preferable to use an absolute path, for example: `/solutions/images/observability-apm-app-landing.png`.

Author: ```suggestion

```

# Images

Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with the following differences:

1. instead of link text, you provide an image description
2. an image description starts with `![` not just `[`
3. there are restrictions on the scope of image paths

:::{note}

> If a page uses a relative image path, it must be relative to the `toc.yml` or `docset.yml` that includes the page rather than relative to the page itself.

Author: ```suggestion

```
